### PR TITLE
Setting up a base for the replying feature

### PR DESCRIPTION
**Pull request by Saransh-cpp**
## Changes/Additions
1. Added `Procfile` and `runtime.txt` for `Heroku` deployment.
2. Refactored the API keys
3. Added the base `Reply` class and a small script to reply to users whenever the bot is mentioned with the hashtag BattBot to test out the deployment

Should be merged after #40

### CODE REVIEW

**Comment by Saransh-cpp** [comment]
The changes would be more clear once #40 is merged

File: bot/tweet_reply.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                # reading the tweet text
+                if '#battbot' in mention.full_text.lower():
+
+                    # replying
+                    api.update_status(
+                        "@"
+                        + mention.user.screen_name
+                        + " Hi there! The replying feature is still under "
+                        + "active development. "
+                        + "Try this again after a few weeks:)",
```
Comment (brosaplanella) [inline bot/tweet_reply.py:72]:
```suggestion
                        + "Try this again in a few weeks :)",
```

Comment (brosaplanella) [reply bot/tweet_reply.py:72]: Very minor style change :)

**Review by brosaplanella — APPROVED** [review]
Looks good, thanks Saransh!

**Pull request merged**